3.0-upg/root/usr/bin/run-mongod

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ${CONTAINER_SCRIPTS_PATH}/common.sh
+
+# Data directory where MongoDB database files live. The data subdirectory is here
+# because mongodb.conf lives in /var/lib/mongodb/ and we don't want a volume to
+# override it.
+export MONGODB_DATADIR=/var/lib/mongodb/data
+
+# Configuration settings.
+export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
+export MONGODB_SMALLFILES=${MONGODB_SMALLFILES:-true}
+export MONGODB_QUIET=${MONGODB_QUIET:-true}
+
+
+export MONGODB_KEYFILE_SOURCE_PATH="/var/run/secrets/mongo/keyfile"
+export MONGODB_KEYFILE_PATH="${HOME}/keyfile"
+
+function usage() {
+  echo "You must specify the following environment variables:"
+  echo "  MONGODB_USER"
+  echo "  MONGODB_PASSWORD"
+  echo "  MONGODB_DATABASE"
+  echo "  MONGODB_ADMIN_PASSWORD"
+  echo "Optional variables:"
+  echo "  MONGODB_REPLICA_NAME"
+  echo "  MONGODB_INITIAL_REPLICA_COUNT"
+  echo "MongoDB settings:"
+  echo "  MONGODB_NOPREALLOC (default: true)"
+  echo "  MONGODB_SMALLFILES (default: true)"
+  echo "  MONGODB_QUIET (default: true)"
+  exit 1
+}
+
+# Make sure env variables don't propagate to mongod process.
+function unset_env_vars() {
+  unset MONGODB_USER MONGODB_PASSWORD MONGODB_DATABASE MONGODB_ADMIN_PASSWORD
+}
+
+function cleanup() {
+  if [ -n "${MONGODB_REPLICA_NAME-}" ]; then
+    mongo_remove
+  fi
+  echo "=> Shutting down MongoDB server ..."
+  if pgrep mongod; then
+    pkill -2 mongod
+  fi
+  wait_for_mongo_down
+  exit 0
+}
+
+echo "
+
+
+Notice: This image is supported only for upgrade from centos/mongodb-26-centos7 to centos/mongodb-32-centos7.
```

Comment:
@omron93 `Notice: This image is supported only for upgrade from MongoDB 2.6 to 3.2.` ?